sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -486,15 +528,16 @@ def update_key_properties(self, name, version=None, **kwargs):
         enabled = kwargs.pop("enabled", None)
         not_before = kwargs.pop("not_before", None)
         expires_on = kwargs.pop("expires_on", None)
-        if enabled is not None or not_before is not None or expires_on is not None:
-            attributes = self._models.KeyAttributes(enabled=enabled, not_before=not_before, expires=expires_on)
-        else:
-            attributes = None
+        attributes = self._get_attributes(enabled=enabled, not_before=not_before, expires_on=expires_on)
```

Comment:
Does the service allow setting `exportable` in an update?

Reply:
Hope you dont mind me chiming in but no - changing the exportable property on updates should not be allowed according to service team. You should be able to update the release policy though of an exportable key